tests/e2e/standard/test_fetch_tables.py

```diff
@@ -160,16 +160,18 @@ def test_fetch_runs_table_by_state(self, environment, project):
         tag = str(uuid.uuid4())
         random_val = random.random()
         with neptune.init_run(project=environment.project) as run:
-            run["sys/tags"] = tag
+            run["sys/tags"].add(tag)
```

Comment:
You can pass the tags via `init_run` :wink:. That would be easier.

Reply:
Good point. I've adjusted